Source/galapagos.cpp

```diff
@@ -1,7 +1,67 @@
 #include "API/galapagos.h"
+
+#include "galapagos_assemblies.h"
+#include "genetic_factory.h"
 #include "population_internal.h"
 #include "stochastic_internal.h"
 
+/****************************
+*****Galapagos Bootstrap*****
+****************************/
+#include <windows.h>
+#include <experimental/filesystem> // TODO: is this still experimental
+namespace fs = std::experimental::filesystem; // namespace alias
+GALAPAGOS_API void gc_initialize() { // TODO: currently hard coded to work against windows
```

Comment:
Explain.

Reply:
gc_initialize kicks off Galapagos' 'bootstrapping' procedure.  It will find all assemblies in the current directory that export the symbol 'gc_bootstrap'. These assemblies will be considered 'plugin' assemblies. gc_initialize will call each plugin assemblies gc_bootstrap function. gc_bootstrap is responsible for registering all plugin objects with the core Galapagos assembly.